src/UnityExtension/Assets/Editor/GitHub.Unity/UI/HistoryView.cs

```diff
@@ -74,6 +74,7 @@ public override void OnEnable()
         {
             base.OnEnable();
             AttachHandlers(Repository);
+            UpdateLog();
```

Comment:
We shouldn't be updating the log here, this gets called every time Unity goes into play mode. The history view should only be loading the log if there's nothing in it, otherwise it should listen for a log update event from the repository and updating the UI only then.